x-pack/plugins/alerting/server/routes/lib/track_legacy_terminology.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { flatten } from 'lodash';
+import { UsageCounter } from 'src/plugins/usage_collection/server';
+
+export const LEGACY_TERMS = ['alertTypeId'];
```

Comment:
I think this is the only term we need to worry about?

Reply:
Are we able to search by action type id? It's a nested field inside the rule SO so it's probably hard to craft a query but we might want to change `actionTypeId` to `connectorTypeId` at some point

Reply:
Yes good point. I forgot about the actions here.

Reply:
I wonder if simply adding `actionTypeId` here is useful? Do you feel the `.includes()` check will properly capture this?